### ISSUE TITLE
Download of OBB?

### ISSUE THREAD

**Issue by Mullen47** [dblstallion/s3eAndroidXAPK#1]
Is this solution suppose to be able to download the obb if its missing, cause I cannot see anywhere this is handled?

**Comment by deemen** [comment]
It doesn't download the OBB if its missing. That must be handled by a separate system. In our game, we used Marmalade's cross-platform HTTP client to download the missing files.

**Comment by Mullen47** [comment]
Ah ok, thought as much. So this basically just resolves the download links? Cheers for the quick reply btw!

**Comment by deemen** [comment]
Correct! Handling the downloading in the extension is a lot of extra work since it would need to be done in Java while Marmalade already has a serviceable downloader in application space.
